internal/extension/smartagentextension/factory.go

```diff
@@ -41,18 +42,26 @@ func NewFactory() component.ExtensionFactory {
 }
 
 var bundleDir = func() string {
-	out := os.Getenv(constants.BundleDirEnvVar)
-	if out == "" {
-		exePath, err := os.Executable()
-		if err != nil {
-			panic("Cannot determine agent executable path, cannot continue")
-		}
-		out, err = filepath.Abs(filepath.Join(filepath.Dir(exePath), ".."))
-		if err != nil {
-			panic("Cannot determine absolute path of executable parent dir " + exePath)
+	dir := os.Getenv(constants.BundleDirEnvVar)
+	if dir == "" {
+		if runtime.GOOS == "windows" {
+			pfDir := os.Getenv("programfiles")
+			if pfDir == "" {
+				pfDir = "C:\\Program Files"
+			}
+			dir = filepath.Join(pfDir, "Splunk", "OpenTelemetry Collector", "agent-bundle")
+			if exePath, err := os.Executable(); err == nil {
+				if colocatedBundle, err := filepath.Abs(filepath.Join(filepath.Dir(exePath), "agent-bundle")); err == nil {
+					if info, err := os.Stat(colocatedBundle); err == nil && info.IsDir() {
+						dir = colocatedBundle
+					}
+				}
+			}
+		} else {
+			dir = filepath.Join("/", "usr", "lib", "splunk-otel-collector", "agent-bundle")
```

Comment:
Not sure how much value we're getting from `filepath.Join` since we're not on Windows in this block, but if we keep it, looks like we could also use `os.PathSeparator` instead of the explicit "/".

Reply:
Updated to just use the unjoined/cleaned string.  I think I've mistaken this practice from the collector as a go convention but there's no real reason for it in this case.